src/libraries/Microsoft.Extensions.Options/src/Microsoft.Extensions.Options.csproj

```diff
@@ -4,6 +4,8 @@
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppPrevious);$(NetCoreAppMinimum);netstandard2.1;netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
     <EnableDefaultItems>true</EnableDefaultItems>
     <IsPackable>true</IsPackable>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <ServicingVersion>1</ServicingVersion>
```

Comment:
LGTM 👍 cc @carlossanlop

Reply:
Congrats @tarekgh, you're the first one to service an 8.0 package :)